src/cli/main.cpp

```diff
@@ -126,28 +126,33 @@ int main(int argc, char* argv[]) {
     experimentalApp
       ->add_option("-I,--include", includeDirs, "Add additional directory to add to front of Ruby $LOAD_PATH (may be used more than once)")
       ->option_text("DIR")
+      ->check(CLI::ExistingDirectory)
```

Comment:
Add some CLI11 validators to ensure the various args are valid directories to begin with and catch mistakes